src/components/Form/From/NumericInput/index.tsx

```diff
@@ -32,28 +33,28 @@ export const NumericInput = ({
   }
 
   return (
-    <div className="flex w-full justify-between">
-      <div className="font-outfit flex-grow text-4xl text-black">
+    <div className="flex justify-between w-full">
+      <div className="flex-grow text-4xl text-black font-outfit">
         <Input
           {...register}
-          autocomplete="off"
-          autocorrect="off"
-          autocapitalize="none"
+          autoComplete="off"
```

Comment:
Where these attributes wrong before and did not have any effect?

Reply:
@TorstenStueber It is because Preact uses raw html attribute names and React not

https://preactjs.com/guide/v10/differences-to-react/#raw-html-attributeproperty-names